compiler/prelude/jsmapping.js

```diff
@@ -236,6 +236,9 @@ var $internalize = (v, t, recv, seen, makeWrapper) => {
         case $kindFloat64:
             return parseFloat(v);
         case $kindArray:
+            if (v === null || v === undefined) {
+                $throwRuntimeError("can not internalize "+v+" as a "+t.string);
```

Comment:
spelling nit ("can not" is not technically wrong, but is definitely less common):

```suggestion
                $throwRuntimeError("cannot internalize "+v+" as a "+t.string);
```

Reply:
I had to laugh at myself when I went to fix this; I noticed that "cannot internalize" is written only a few lines above and below my new one. 🤦 I'm not great at English even though it's my only language. 😆